zgl.zig

```diff
@@ -1023,7 +1053,7 @@ pub const TextureParameter = enum(c.GLenum) {
     wrap_r = c.GL_TEXTURE_WRAP_R,
 };
 
-fn TextureParameterType(comptime param: TextureParameter) type {
+pub fn TextureParameterType(comptime param: TextureParameter) type {
```

Comment:
not sure if pub is necessary, maybe I missed another way to set the textureparametertype argument of textureParameter()?

Reply:
afaik it should not be necessary, as the TextureParameterType is determined by the parameter you want to set and is imho not required to be computed generically outside the module.